src/components/ATATPhoneInput.vue

```diff
@@ -232,18 +274,30 @@ export default class ATATPhoneInput extends Vue {
       const phoneTextField = document.getElementById(
         this.id + "_textField"
       ) as HTMLElement;
-    
-      return Inputmask({ 
+
+      return Inputmask({
         mask: this._selectedCountry?.mask || [],
-        placeholder: "", 
-        jitMasking: true 
+        placeholder: "",
+        jitMasking: true
       }).mask(phoneTextField);
     });
   }
-  
+  private setExtensionMask(): void{
+    Vue.nextTick(()=>{
+      const extensionTextField = document.getElementById(
+        this.id + "_PhoneExtension_text_field"
+      ) as HTMLElement;
+      return Inputmask({
+        regex:"^[0-9]{1,6}?",
+        placeholder: "",
+        jitMasking: true
+      }).mask(extensionTextField);
+    });
+  }
   private mounted(): void {
     this.searchResults = [...this.countries];
     this.setPhoneMask();
+    this.setExtensionMask();
```

Comment:
- [ ] more likely not to give stooopid & usually harmless console errors if the `setExtensionMask` is only called if there is a phone extension.
`if (phoneExtension) { this.setExtensionMask();}`